test/fixtures/typescript/nuxt.config.js

```diff
@@ -1,3 +1,6 @@
 export default {
+  build: {
+    useForkTsChecker: true
```

Comment:
Approve as a non breaking change as 2.4 is not released.

Reply:
There is no changes to do around here right ?

Reply:
Yeah. Commented to confirm this change. 🙈